src/docs_website/src/docs.zig

```diff
@@ -371,7 +371,12 @@ const Page = struct {
         docs: *std.Build.Step.WriteFile,
         search_index: *SearchIndex,
     ) !void {
+        const vale_step = std.Build.Step.Run.create(b, "run vale");
+        vale_step.addFileArg(website.vale_bin);
+        vale_step.addFileArg(b.path(self.path_source));
```

Comment:
Hm, but this won't run on Changeling, and other things that are not part of the docs, right?

Maybe we should rather have a top-level step which runs vale on the root of the repo for all markdown files?

Reply:
True, that is the changelog, readme and internal docs I think (everything not part of the docs website).

I also noticed that Vale is quite slow when running on individual files (it's a large executable). So maybe I implement the spellcheck as a build option.